src/tools/r2rdump/R2RDump.csproj

```diff
@@ -14,11 +14,16 @@
 	<TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
 	<CLSCompliant>false</CLSCompliant>
 	<NoWarn>8002,NU1701</NoWarn>
```

Comment:
Nit: The indenting looks weird in this file. Would you mind converting any tabs to spaces so it lines up nicely.